METADATA.yaml

```diff
@@ -0,0 +1,22 @@
+title: Infrastructure
+description: wg-infra Working Group is responsible for AMP's infrastructure, including building, testing and release.
+facilitator:
+  login: rsimha
+  name: Raghu Simha
+members:
+  - login: danielrozenberg
+    name: Daniel Rozenberg
+  - login: erwinmombay
+    name: Erwin Mombay
+  - login: estherkim
+    name: Esther Kim
```

Comment:
Could you please add @rcebulko to this list?

Reply:
Done! :)

Reply:
:tada:

Reply:
Can you please merge?